product_shelf_management/report/report_deliveryslip.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="report_delivery_document" inherit_id="stock.report_delivery_document">
+        <xpath expr="//span[@t-field='move.product_id']" position="before">
+            <t>保管: <span t-field="move.product_id.shelf_id" /></t>
```

Comment:
Why don't we stick to the practice of having English as the source string and add the translation?

Reply:
OK, I have adjusted to the standard practice.